onnxruntime/contrib_ops/rocm/diffusion/group_norm_impl.cu

```diff
@@ -37,18 +38,23 @@ Status LaunchGroupNormKernel(
                            "only num_groups=32 is supported. Got", num_groups);
   }
 
-  GroupNormNHWCParams<T> params(nullptr, stream, output, reinterpret_cast<float*>(workspace), input, gamma, beta,
+  GroupNormNHWCParams<T> params(tuning_ctx, stream, output, reinterpret_cast<float*>(workspace), input, gamma, beta,
                                 batch_size, height, width, num_channels, num_groups, epsilon, use_swish_activation);
 
+  if (tuning_ctx != nullptr && tuning_ctx->IsTunableOpEnabled()) {
```

Comment:
Do we still encounter `tuning_ctx == nulptr`?

Reply:
It will not be encountered here, I will remove it.